### PR TITLE
Intro to C++ - Part 2 (If Else Loops)

### PR DESCRIPTION
**Pull request by param31**
The second article in the C++ series talks about If else and Loops.

### CODE REVIEW

**Comment by hectorkambow** [comment]
@nadivgold - good morning 👋  Nadiv - If you could please give this article a review - I have added the front matter. (Will be adding the front matter) doing the plagiarism check(s) and the Hugo previewing on our end. 
Please let me know if you have questions along the way 🚀

**Comment by hectorkambow** [comment]
Plagiarism Check done

File: articles/Intro to C++ - Part 2/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+     }
+     else{
+      cout<<"Ohh! You failed"<<endl;
+     }
+     
+  return 0;
+  }
+  ```
+
+This program will print "You passed" if your score is more than 60 and "You failed" if your score is less than 60.
```
Comment (nadivgold) [inline articles/Intro to C++ - Part 2/index.md:47]: Greater than or equal to

Comment (param31) [reply articles/Intro to C++ - Part 2/index.md]: Just resolved it.

File: articles/Intro to C++ - Part 2/index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+* While loop
+* For loop
+* Do while loop
+
+But most of the time, we will be using **While** and **For Loop** only. Let's see what the differences are between them and how they are different from **If-Else** statements.
+
+**While Loop** 
+
+There is a True-False condition at the top. We only enter the block if the test is true, and do the whole block only and if the test is true.
```
Comment (nadivgold) [inline articles/Intro to C++ - Part 2/index.md:93]: There is a True-False condition at the top. We only enter the block if the test is true, and do the whole block only if the test is true.

Comment (param31) [reply articles/Intro to C++ - Part 2/index.md]: Resolved.

**Comment by hectorkambow** [comment]
@ninjaginja 
Molly if you could give this article a quick review when you get a chance. 
I have also given it a preliminary review for readability and formatting & I've had Navid give it a tech review as well.
I just wanted to get a final review before we move forward. 
Thank you 🚀

**Comment by hectorkambow** [comment]
@param31 Please go over the changes made for grammar, readability, and incorrect code outputs to the final version to better incorporate those types of changes into your future articles. 🚀
Thank you v much

**Comment by hectorkambow** [comment]
@nadivgold -to strengthen up our review skills - let's go over the final version of this article to see where we could improve and see  the level of content were striving for from the students 🚀  
Thank you

**Comment by nadivgold** [comment]
@hectorkambow took a look over the changes. Looks good. Glad molly was there to catch some of the things that slipped past me.

**Pull request merged**